models/equaler.go

```diff
@@ -5,4 +5,22 @@ type Equaler interface {
 	// Returns true if the Equaler object is the same as this object; otherwise false is returned.
 	// You need to convert the Equaler object using type assertions: https://golang.org/ref/spec#Type_assertions
 	Equal(Equaler) bool
-}
\ No newline at end of file
+}
+
+// DummyEqualer implements the Equaler interface and can be used by tests.
+// Other than that it has not meaning.
+type DummyEqualer struct {
```

Comment:
It is used in all the tests from different packages to test the type difference of two `Equaler` implementations. See [here](https://github.com/almighty/almighty-core/pull/240/files#diff-649fa21e6bc590dce895de2d3607470dR25) for example.